uptobox.sh

```diff
@@ -87,7 +87,7 @@ uptobox_cloudflare() {
 # stdout: real file download link
 uptobox_download() {
     local -r COOKIE_FILE=$1
-    local -r URL=$(replace '://www.' '://' <<< "$2")
+    local -r URL=$(replace '://www.' '://' <<< "$2" | replace 'http:' 'https:')
     local -r BASE_URL='http://uptobox.com'
```

Comment:
What about changing BASE_URL and use https ?

Reply:
BASE_URL is only used to guess what type of account it is. (line 99)